### PR TITLE
Us01 sign up form

### PR DESCRIPTION
**Pull request by Gumatan**

### CODE REVIEW

File: frontend/src/components/SignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      </div>
+      <div className="connect">
+        <h2>
+          <span>Vous avez</span> un compte ?
+        </h2>
+        <form onSubmit={handleSubmit}>
+          <h3>Identifiant</h3>
+          <input type="text" />
+          <h3>Mot de passe</h3>
+          <input type="text" />
```
Comment (Adwarf14) [inline frontend/src/components/SignIn.jsx]: type = "password"

**Review by Adwarf14 — DISMISSED** [review]

File: frontend/src/components/style/variables.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// COLORS
+$mainPurple: #8d20ae;
```
Comment (LoicBrassart) [inline frontend/src/components/style/variables.scss]: I'd advice against using color names in variable names : if your client wants to transition the main color to YellowGreen ( <3 ), you'll have something like `$mainPurple: yellowgreen`... ಠ_ಠ

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- No fonts outside the `public` folder

**Review by Adwarf14 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**